content/docs/1.0.0/tutorial/manage-your-microservices.md

```diff
@@ -2,79 +2,97 @@
 </aside>
 
-This tutorial includes 5 microservices already running. We can view any configured microservices with `microservice list`:
+This tutorial includes 3 microservices already running. We can view any configured microservices using Controller CLI: `microservice list`.
 
-```sh
-iofog-controller microservice list
+```bash
+docker exec -ti iofog-controller iofog-controller microservice list
 ```
 
 This returns a JSON object containing the full list, along with their configuration. The most important microservices currently running are:
 
-###### Sensors
+###### Tutorial Microservices
 
-A microservice that pretends to be reading data from a local hardware sensor. The data it produces is published with [the SDK](../writing-microservices/sdk.html) and routed through the [Connector](../connectors/overview.html) so that it can be read by other microservices.
+The tutorial consists of three microservices deployed on top of ioFog stack.
 
-[Source Code](https://github.com/ioFog/example-microservices/tree/master/sensors-data)
+The *Sensors* microservice pretends to be reading data from a local hardware sensor. The data it produces is published with [the SDK](../writing-microservices/sdk.html) and routed through the [Connector](../connectors/overview.html) to the REST API microservice, so that it can be read by other microservices that only understand REST API.
 
-###### API
+[Sensors microservice source code on Github](https://github.com/ioFog/example-microservices/tree/master/sensors-data)
 
-A generic microservice that provides a REST API web server, allowing access to any arbitrary data source connected using the Controller.
+The *REST API* is a generic microservice that provides a REST API web server, allowing access to any arbitrary data source connected using the Controller.
 
-[Source Code](https://github.com/ioFog/example-microservices/tree/master/json-rest-api-cors-enabled)
+[REST API microservice source code on Github](https://github.com/ioFog/example-microservices/tree/master/json-rest-api-cors-enabled)
 
-###### Freeboard
-
-Another generic web server microservice that provides an HTML dashboard to view the real-time results coming from the API data source.
+*Freeboard* is the last microservice that provides an HTML dashboard to view the real-time results coming from a rest API data source. In the case of our tutorial, the source of the data is our RESP API microservice.
 
 ## Routes
 
 The Sensors and REST API microservices are generic; they are not hardcoded to talk with each other, instead the relationship dictating the flow of data was configured with the Controller. This is in the spirit of the microservice architecture, separating concerns into pieces so that you can combine and interchange them.
```

Comment:
The Sensors and REST API microservices are generic**.** **They** are not hardcoded to talk with each other, instead**,** the relationship dictating the flow of data was configured with the Controller.

you > we